### PR TITLE
Move building and publishing lambda-utils package into its own project

### PR DESCRIPTION
**Pull request by kenoir**

### CODE REVIEW

File: lambda_utils/Makefile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+		--env FIND_MATCH_PATHS="/data" \
+		python3.6_ci:latest
+
+lambda_utils-build:
+	docker run \
+		--volume $(LAMBDA_UTILS):/src \
+		--workdir /src \
+		python:3-alpine python setup.py sdist
+
+lambda_utils-publish: lambda_utils-build
```
Comment (kenoir) [inline lambda_utils/Makefile]: This should be `deploy`

Comment (alexwlchan) [reply lambda_utils/Makefile]: Mmm. It’s publishing a new version of something to a package repository, but not actually deploying it anywhere in our stack. I prefer "publish", a la ECR publishing.

Comment (kenoir) [reply lambda_utils/Makefile]: It should be publish, but our build system relies on this being "deploy".

Comment (kenoir) [reply lambda_utils/Makefile]: No, strike that run_task.py says otherwise.

**Pull request merged**